README.md

```diff
@@ -190,6 +190,7 @@ This parameters could be included in a separate file, like `dataset-config.json`
 - `bid_inv` (optional): String or array of strings with the bid invitations. Bids are used by the reviewers in OpenReview to select papers that they would or would not like to review. These bids are then used to compute a final affinity score to be more fair with the reviewers.
 - `use_email_ids` (optional): Boolean value. If true, then the email of the user is used instead of his/her OpenReview Profile ID.
 - `max_workers` (optional): Number indicating the amount of workers that will be used to retrieve the expertise from OpenReview. If it is not set, it will use the maximum available workers of your machine by default. The more workers the faster the creation of the dataset
+- `version` (optional): An integer indicating the version of the OpenReview API to use
```

Comment:
mention here that default is set to 1

Reply:
yes, please version 1 should be the default one